src/main/pushNotifications/PushNotificationProvider.tsx

```
@@ -0,0 +1,146 @@
+  useEffect(() => {
+    if (!pushState.isSupported || !pushState.isServerEnabled) return;
+    if (Notification.permission !== 'granted') return;
+
+    const silentRefresh = async () => {
+      try {
+        const registration = await navigator.serviceWorker.ready;
+        const subscription = await registration.pushManager.getSubscription();
+
+        if (!subscription) {
+          // No browser subscription — silently re-subscribe
+          const vapidResponse = await fetch('/api/private/non-interactive/graphql', {
```

Biome (2.4.7)

[error] This variable vapidResponse is unused.

(lint/correctness/noUnusedVariables)